tests/hipify-clang/axpy.cu

```diff
@@ -1,9 +1,9 @@
-// RUN: hipify "%s" -o=%t -- %cuda_args
+// RUN: hipify "%s" -o=%t -- %cuda_args && cat %t | sed -Ee 's|//.+|// |g' | FileCheck %s
```

Comment:
One thing we might like to do later is make `hipify` use clang's normal output specification options. This'll make kludges like this one unnecessary, which would be nice. (If we had a way to output to stdout, this line could lose the `&& cat %t` and the `-o=%t` parts.)

The `sed` part is just to make failures nicer. We have to strip the `CHECK` directives from the input file, otherwise all the tests will triviall pass by matching with their own definitions. This sed line does that, leaving the `//` in, so the line-count is unchanged, and so the line number in error messages is correct.